lib/remote/apilistener-configsync.cpp

```diff
@@ -104,7 +110,32 @@ Value ApiListener::ConfigUpdateObjectAPIHandler(const MessageOrigin::Ptr& origin
 		return Empty;
 	}
 
+	auto isObjectLocked = [&ptype, &objName] {
+		auto type = ApiListener::m_ObjectNames.find(ptype.get());
+		if (type == ApiListener::m_ObjectNames.end()) {
```

Comment:
I'd capture _this_ instead of using `ApiListener::`.

Reply:
That's a static method there's no `this`. `ApiListener::` is just some leftover from the previous version